frontend/packages/operator-lifecycle-manager/src/components/modals/uninstall-operator-modal.tsx

```diff
@@ -58,6 +58,15 @@ export const UninstallOperatorModal = withHandlePromise((props: UninstallOperato
       .catch(_.noop);
   };
 
+  const uninstallMsg =
+    props.subscription.metadata.namespace === 'openshift-operators' ? (
+      'all the installed namespaces'
```

Comment:
```suggestion
      'all namespaces'
```

We might want to bold this text or make it stronger somehow.